File: data/imported/knowledge/slides/net/merged/snippets/snippet_023.cs
[Fact]
    public void TestCustomIntProperty()
    {
        // Custom integer properties persist.
        using var pres = new Presentation();
        pres.DocumentProperties.SetCustomPropertyValue("Count", 42);

        using var pres2 = TestHelpers.SaveAndReopen(pres, _tempDir);
        var value = pres2.DocumentProperties.GetCustomPropertyValue("Count");
        value.Should().Be(42);
    }

File: data/imported/knowledge/slides/java/merged/snippets/snippet_023.java
@Test
    void testCustomIntProperty() throws IOException {
        try (var pres = new Presentation()) {
            pres.getDocumentProperties().setCustomPropertyValue("Count", 42);

            try (var pres2 = saveAndReopen(pres)) {
                List<Object> out = new ArrayList<>();
                out.add(null);
                pres2.getDocumentProperties().getCustomPropertyValue("Count", out);
                assertThat(out.get(0)).isEqualTo(42);
            }
        }
    }